.github/workflows/pr-check-lint_content.yml

```
@@ -119,7 +119,7 @@ jobs:
 
           echo "Running markdownlint --fix"
           MD_LINT_FAILED=false
-          MD_LINT_LOG=$(yarn markdownlint-cli2 --fix "${files_to_lint[@]}" 2>&1) || MD_LINT_FAILED=true
+          MD_LINT_LOG=$(npx markdownlint-cli2 -- --fix "${files_to_lint[@]}" 2>&1) || MD_LINT_FAILED=true
```

npx doesn't need `--`.

```suggestion
          MD_LINT_LOG=$(npx markdownlint-cli2 --fix "${files_to_lint[@]}" 2>&1) || MD_LINT_FAILED=true
```